tests/test_simulation.py

```diff
@@ -1615,6 +1748,224 @@ def test_elastic_strain_eqv_von_mises_elemental(self, modal_simulation):
         assert field.component_count == 1
         assert np.allclose(field.data, field_ref.data)
 
+    def test_disp_external_layer(
+        self, frame_modal_simulation: post.ModalMechanicalSimulation
+    ):
+        result = frame_modal_simulation.displacement(set_ids=[1], external_layer=True)
+        result_all = frame_modal_simulation.displacement(
+            set_ids=[1], external_layer=False
+        )
+        assert len(result.index.mesh_index) == 5886
+        assert len(result.index.mesh_index) == len(result_all.index.mesh_index)
```

Comment:
Why would we necessarily have the same number of nodes, this is due to this specific case, right?

Reply:
Yes the model is small enough that the external layer is everything